builtin/logical/transit/path_cache_config.go

```diff
@@ -51,22 +42,21 @@ This can be a long operation for large syncmap caches.
 }
 
 func (b *backend) pathCacheConfigWrite(ctx context.Context, req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
-
 	// get target cacheType
 	cacheTypeStr := d.Get("cache-type").(string)
 	cacheSize := d.Get("cache-size").(int)
 	var cacheType keysutil.CacheType
 	switch cacheTypeStr {
 	case "syncmap":
-		cacheType = keysutil.SYNCMAP
+		cacheType = keysutil.SyncMap
 	case "lru":
 		cacheType = keysutil.LRU
 	default:
-		cacheType = keysutil.NOTIMPLEMENTED
+		cacheType = keysutil.NotImplemented
 	}
 
 	// err if the requested cacheType has not been implemented
-	if cacheType == keysutil.NOTIMPLEMENTED {
+	if cacheType == keysutil.NotImplemented {
 		return logical.ErrorResponse(fmt.Sprintf("unknown cache-type %s", cacheTypeStr)), logical.ErrInvalidRequest
```

Comment:
Minor nit: use %q instead of %s, will be more readable e.g. in the empty string case.